main.cpp

```diff
@@ -53,83 +53,114 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 //  1)
 //  Noun: synth
-//  action 1: synth.playNote();
+//  action 1: The synth plays the note that's pressed on the keyboard.
+synth.playNote();
 
-//  action 2: synth.arpeggiateNotes();
+//  action 2: The synth activates its arpeggiating function.
+synth.arpeggiateNotes();
 
-//  action 3: synth.modulateEffect();
+//  action 3: A synth parameter modulates an effect.
+synth.modulateEffect();
 
 //  2)
 //  Noun: audioPlayer
-//  action 1: audioPlayer.playAudio();
+//  action 1: The audio player starts playing the current file.
+audioPlayer.playAudio();

```

Comment:
The pseudo-code does not match the plain-english action: `playCurrentFile();`
- [ ] revise